story/css/index.html

```diff
@@ -1 +1,26 @@
-<script>window.location.replace("http://xmas.sitepoint.com");</script>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+
+    <meta property="og:title" content="SitePoint Christmas Sale - CSS" />
+    <meta property="og:type" content="website" />
+    <meta
+      property="og:image"
+      content="https://xmas.sitepoint.com/img/logo_small.png"
+    />
+    <meta property="og:site_name" content="SitePoint Christmas Sale - CSS" />
+    <meta property="og:description" content="SitePoint Christmas Sale - CSS" />
+
+    <title>SitePoint Christmas Sale - CSS</title>
+    <meta name="description" content="SitePoint Christmas sale website - CSS" />
+    <meta name="title" content="SitePoint Christmas Sale -CSS" />
+    <meta name="viewport" content="width=device-width" />
+  </head>
+  <body>
+    <script>
+      window.location.replace('http://xmas.sitepoint.com');
```

Comment:
I'm glad the SEO issues can be resolved in this manner, however I believe this URL should use https.

Reply:
Updated to `https`. Thanks!